src/wp-includes/media.php

```diff
@@ -5789,6 +5808,18 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 		}
 	}
 
+	/**
+	 * Filters the loading optimization attributes.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param array  $loading_attrs The loading optimization attributes.
+	 * @param string $tag_name      The tag name.
+	 * @param array  $attr          Array of the attributes for the tag.
+	 * @param string $context       Context for the element for which the loading optimization attribute is requested.
+	 */
+	$loading_attrs = apply_filters( 'wp_get_loading_optimization_attributes', $loading_attrs, $tag_name, $attr, $context );
+
 	return $loading_attrs;
```

Comment:
This filter looks good, however we will also need to apply it everywhere else further above where there are early returns (except if the "pre-" filter was used). In those instances, you can refer to the filter docs here with the typical `/** This filter is documented in wp-includes/media.php */` annotation.

Small nit-pick, we can avoid the variable:
```suggestion
	return apply_filters( 'wp_get_loading_optimization_attributes', $loading_attrs, $tag_name, $attr, $context );
```

Reply:
Good point in the early bails.
Having the filter being applied to a variable, comes from a personal preference, as it's easy to add a breakpoint and see if the filter is changing the value.